channels/chan_usbradio.c

```diff
@@ -4933,6 +4938,7 @@ static struct chan_usbradio_pvt *store_config(const struct ast_config *cfg, cons
 		CV_UINT("txhpf", o->txhpf);
 		CV_UINT("sendvoter", o->sendvoter);
 		CV_UINT("clipledgpio", o->clipledgpio);
+		CV_BOOL("legacyaudioscaling", o->legacyaudioscaling);
```

Comment:
What do you think about only updating the "legacyaudioscaling" variable in the .conf file when enabled? Asking because I'm thinking about the future when the default value would be not enabled and when there's no need to call attention to the value.